client/verta/verta/dataset/_path.py

```diff
@@ -103,6 +103,9 @@ def _from_proto(cls, blob_msg):
                 for component_msg in blob_msg.dataset.path.components
             ]
         )
+        obj._mdb_versioned = any(
+            component._internal_versioned_path for component in obj.list_components()
+        )
```

Comment:
This is a long story, but basically when `enable_mdb_versioning=True`, the client will compute an `_internal_versioned_path` for each captured file. Otherwise, that field is empty.